journal/vendor-extra/Views/ViewConverter/ViewConverterRegistry.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Libero\Views\ViewConverter;
+
+use Libero\Dom\Node;
+use Libero\Views\View;
+use Libero\Views\ViewConverter;
+
+final class ViewConverterRegistry implements ViewConverter
+{
+    private $registry = [];
+
+    public function add(ViewConverter $registry)
+    {
+        $this->registry[] = $registry;
+    }
+
+    public function convert(Node $object, ?string $template, array $context = []) : ?View
+    {
+        $view = null;
+
+        foreach ($this->registry as $converter) {
+            $converted = $converter->convert($object, $template, $context);
+
+            if (!$view instanceof View) {
+                $view = $converted;
+            } elseif ($converted instanceof View) {
+                $view = $view->withArguments($converted->getArguments())->withTemplate($converted->getTemplate());
```

Comment:
This allows the results of `eLifeFrontContentHeaderConverter` and `FrontContentHeaderConverter` to be combined.

Might want to think about prioritisation, to allow changing of arguments. (Currently ambitious as to what order they run in.)

Reply:
Could maybe pass an empty `View` instead of `$template` and let them populate it as they go along.

Reply:
Related to https://github.com/libero/walking-skeleton/pull/57#discussion_r208680885, these should be separated. So delegating the actual conversion to visitors, that take a `View` and add details as necessary.